src/rocksdb.rs

```diff
@@ -512,6 +512,70 @@ impl DB {
         }
     }
 
+    /// Query a RocksDB instance for a "property" that returns a numeric value. The available
+    /// properties are listed
+    /// [here](https://github.com/facebook/rocksdb/blob/fefd4b98c572a09da47fb100da9e1472d28c0d08/include/rocksdb/db.h#L522)
+    /// but note that only properties returning a numeric value can be queried using this function.
+    pub fn get_int_property(&self, prop: &str) -> u64 {
+        let mut value = 0u64;
+        let prop = CString::new(prop).expect("Rust &str is checked by the compiler");
+        unsafe {
+            rocksdb_ffi::rocksdb_property_int(
+                self.inner,
+                prop.as_ptr(),
+                &mut value,
+            );
+        }
+        value
+    }
+
+    /// Query a RocksDB instance scoped to a column family for a "property" that returns a numeric
+    /// value. The available properties are listed
+    /// [here](https://github.com/facebook/rocksdb/blob/fefd4b98c572a09da47fb100da9e1472d28c0d08/include/rocksdb/db.h#L522)
+    /// but note that only properties returning a numeric value can be queried using this function.
+    pub fn get_int_property_cf(&self, cf: Column , prop: &str) -> u64 {
+        let string_value = self.get_property_value_cf(cf, prop);
+        string_value.parse::<u64>().unwrap_or(0)
```

Comment:
maybe add a comment that this is a tmp workaround for https://github.com/facebook/rocksdb/issues/5249 (or create an issue)

Reply:
I noticed today that upstream `rust-rocksdb` does this too: https://github.com/rust-rocksdb/rust-rocksdb/blob/184440b2095979baeb747ccf300022198e9606ed/src/db.rs#L1529

Working on a proper patch in facebook/rocksdb.